### PR TITLE
Update roadmap

### LINKED ISSUE #572: Add 2018 to Roadmap page
Add 2018 to Roadmap page and move over the ongoing tasks from 2017.

### PR DESCRIPTION
**Pull request by mattcode55**
Fix #572

### CODE REVIEW

**Comment by mattcode55** [comment]
Sorry for the big diff, the formatting on the roadmap page made me cry and then my editor deleted the trailing spaces from custom.css.

File: _data/roadmap.yml [hunk 1]
```diff
@@ -78,32 +78,28 @@
...
     - name: Multi-signatures (multisig)
+      date: December, 2017
+      status: completed
+- year: 2018
+  accomplishments:
+    - name: Forum Funding System redesign
       date:
       status: ongoing
-    - name: Kovri alpha release
+    - name: More efficient range proofs for RingCT to reduce transaction sizes
```
Comment (leonklingele) [inline _data/roadmap.yml]: Are we talking about Bulletproofs? Might be worth mentioning it in this item.

File: css/custom.css [hunk 2]
```diff
@@ -1068,41 +1068,41 @@ pre.highlight>code {
...
+  width: 0;
+  height: 0;
   border-left: 3px solid transparent;
   border-right: 3px solid transparent;
   border-top: 3px solid #393939;
 }
 
 @media only screen and (max-width: 75rem) {
-    
+
```
Comment (leonklingele) [inline css/custom.css]: While I absolutely agree with removing trailing spaces, it's apparently not something greatly appreciated (especially in this case as so many unrelated lines are updated).

File: resources/roadmap/index.md [hunk 3]
```diff
@@ -3,145 +3,147 @@ layout: custom
...
-       </div>
-       <div class="col-xs-4">
-            <li class="ongoing"></li>
-            <p>Ongoing task</p>
-       </div>
-       <div class="col-xs-4">
-            <li class="upcoming"></li>
-            <p>Upcoming task</p>
-       </div>
+    <div class="text-center pre-roadmap container">
```
Comment (leonklingele) [inline resources/roadmap/index.md]: Hmm, better fix indentation in a dedicated PR. This just makes it nearly impossible to review.

File: _data/roadmap.yml (hunk 1 above)
Comment (mattcode55) [reply _data/roadmap.yml]: I just moved the old items over, but yes I think that it references bulletproofs.

**Comment by mattcode55** [comment]
@leonklingele Diff isn't as noisy now.

File: resources/roadmap/index.md [hunk 4]
```diff
@@ -112,13 +112,35 @@ title: Roadmap
                               
                               {%endfor%}
                             </div>
-                            <label for="tab-5" tabindex="0"></label>
-                            <input id="tab-5" type="radio" name="tabs" aria-hidden="true">
+			    <label for="tab-5" tabindex="0"></label>
```
Comment (leonklingele) [inline resources/roadmap/index.md]: Please maintain space indentation.

**Comment by mattcode55** [comment]
Thanks, fixed.

File: resources/roadmap/index.md [hunk 5]
```diff
@@ -113,12 +113,34 @@ title: Roadmap
                               {%endfor%}
                             </div>
                             <label for="tab-5" tabindex="0"></label>
-                            <input id="tab-5" type="radio" name="tabs" aria-hidden="true">
+                            <input id="tab-5" type="radio" name="tabs" aria-hidden="true" checked>
+                            <h2>2018</h2>
+                            <div class="tabPanel-content">
+                              {% for toplevel in site.data.roadmap %} {% if toplevel.year == 2018 %} {% for roadlist in toplevel.accomplishments %}
+                              <div class="row start-xs">
```
Comment (leonklingele) [inline resources/roadmap/index.md]: Any reason this no longer has the `middle-xs` class? lgtm otherwise :)

File: _data/roadmap.yml [hunk 6]
```diff
@@ -78,32 +78,28 @@
     - name: 0MQ/ZeroMQ
       date: September, 2017
       status: completed
-    - name: Fluffy blocks
```
Comment (MaxXor) [inline _data/roadmap.yml]: Why remove this? Would be better if marked as completed imho.

Comment (mattcode55) [reply _data/roadmap.yml]: It was a duplicate entry, Fluffyblocks were already marked as completed above.

https://github.com/mattcode55/monero-site/blob/30300c95661dea9fec172337863e861e1423692a/_data/roadmap.yml#L69-L70

Comment (MaxXor) [reply _data/roadmap.yml]: Oh okay, thanks!

**Comment by QuickBASIC** [comment]
For Future should we add the bootstrapping daemon mode that was supposed to help with new wallet onboarding? Is that still planned to be implemented eventually?

**Comment by mattcode55** [comment]
Rebased on rehrar's translation update.

***

@QuickBASIC: Not sure, but we can easily add it if it is still planned

**Comment by rehrar** [comment]
Anything else needed on this before merge? @mattcode55

**Comment by mattcode55** [comment]
Sorry @rehrar, I must've missed this comment before. I think everything is good and it's ready to be merged.

**Comment by mattcode55** [comment]
ping @rehrar

**Comment by el00ruobuob** [comment]
Tested on the following platforms:

  | Edge | IE 11 | Chrome | Firefox | Opera | Vivaldi
-- | -- | -- | -- | -- | -- | --
Windows 7 | - | OK | OK | OK | OK | OK
Windows 10 | OK | OK | OK | OK | OK | OK
Ubuntu 18.04 | - | - | OK | OK | OK | OK
CentOS 7 | - | - | - | OK | OK | OK

@rehrar @luigi1111 could you merge?

**Comment by rehrar** [comment]
Yeesh, Ubuntu is on 18.04 already. If it looks good in all those things, I see no reason for this not to be merged. Thank you for helping check that for me.

**Comment by mattcode55** [comment]
Thanks for the testing @el00ruobuob 

Maybe we could set up a bot that automatically screenshots affected pages and posts the screenshots as a comment to the PR, to make this kind of testing easier.
